### PR TITLE
[REVIEW] fix/ci-cupy

### LINKED ISSUE #213: CI: cupy tests skipped
The cupy tests are [skipped](https://gpuci.gpuopenanalytics.com/blue/organizations/jenkins/rapidsai%2Fgpuci-cuda101%2Fucx-py%2Fprb%2Fucx-py-gpu-build/detail/ucx-py-gpu-build/73/pipeline#log-2413) even though it is installed here: 
https://github.com/rapidsai/ucx-py/blob/branch-0.10/ci/gpu/build.sh#L49

Any ideas what is wrong?

cc. @quasiben @jakirkham

### PR DESCRIPTION
**Pull request by quasiben**
resolves #213

### CODE REVIEW

**Comment by quasiben** [comment]
CuPy tests are passing:

```
tests/test_send_recv.py::test_send_recv_cupy[f8-1] PASSED                [ 61%]

tests/test_send_recv.py::test_send_recv_cupy[f8-16] PASSED               [ 63%]

tests/test_send_recv.py::test_send_recv_cupy[f8-256] PASSED              [ 64%]

tests/test_send_recv.py::test_send_recv_cupy[f8-4096] PASSED             [ 65%]

tests/test_send_recv.py::test_send_recv_cupy[f8-65536] PASSED            [ 67%]

tests/test_send_recv.py::test_send_recv_cupy[f8-1048576] PASSED          [ 68%]

tests/test_send_recv.py::test_send_recv_cupy[f8-16777216] PASSED         [ 69%]
```

But the cuDF tests are not being run and i'm trying to figure that out.

**Comment by quasiben** [comment]
Great!  Everything seems to be working now.

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -45,15 +45,18 @@ nvidia-smi
...
+              "dask>=2.3.0" "distributed>=2.3.2" "numpy>=1.16" -c rapidsai-nightly
 
 # needed for asynccontextmanager in py36
 conda install -c conda-forge "async_generator" "automake" "libtool" \
                               "cmake" "automake" "autoconf" "cython" \
                               "pytest" "pkg-config" "pytest-asyncio"
 
 
+# install cupy
+conda install -c rapidsai -c nvidia "cupy>=6.3.0"
```
Comment (jakirkham) [inline ci/gpu/build.sh]: Is there a reason it needs to be CuPy 6.3.0? Or would CuPy 6.2.0 work?

Comment (quasiben) [reply ci/gpu/build.sh]: cupy 6.20 should also be fine.  I'll get a PR in late today

Comment (jakirkham) [reply ci/gpu/build.sh]: Added PR ( https://github.com/rapidsai/ucx-py/pull/239 ).

**Pull request merged**